pkg/daemon/ceph/osd/volume.go

```diff
@@ -137,7 +163,24 @@ func getLVPath(op string) string {
 	return ""
 }
 
-func updateLVMConfig(context *clusterd.Context, onPVC bool) error {
+func getLVFromDevicePath(context *clusterd.Context, devicePath string) (string, error) {
+	devInfo, err := context.Executor.ExecuteCommandWithOutput(true, "",
+		"dmsetup", "info", "-c", "--noheadings", "-o", "name", devicePath)
+	if err != nil {
+		return "", fmt.Errorf("failed dmsetup info. output: %s, err: %v", devInfo, err)
+	}
+	out, err := context.Executor.ExecuteCommandWithOutput(true, "", "dmsetup", "splitname", devInfo, "--noheadings")
+	if err != nil {
+		return "", fmt.Errorf("failed dmsetup splitname %s. err: %v", devInfo, err)
+	}
+	split := strings.Split(out, ":")
+	if len(split) < 2 {
+		return "", fmt.Errorf("dmsetup splitname returned unexpected result for %s. output: %s", devInfo, out)
```

Comment:
use: `fmt.Errorf("dmsetup splitname returned unexpected result for %q. output: %q", devInfo, out)`